project.clj

```diff
@@ -213,12 +225,6 @@
   :deploy-repositories [["releases" ~(deploy-info "https://artifactory.delivery.puppetlabs.net/artifactory/list/clojure-releases__local/")]
                         ["snapshots" ~(deploy-info "https://artifactory.delivery.puppetlabs.net/artifactory/list/clojure-snapshots__local/")]]
 
-  ;; By declaring a classifier here and a corresponding profile below we'll get an additional jar
-  ;; during `lein jar` that has all the code in the test/ directory. Downstream projects can then
-  ;; depend on this test jar using a :classifier in their :dependencies to reuse the test utility
-  ;; code that we have.
-  :classifiers  [["test" :testutils]]
```

Comment:
Is this going to break compatibility with pe extensions? https://github.com/puppetlabs/pe-puppetdb-extensions/blob/master/project.clj#L88

Reply:
Umm, how about I actually test this pr against pdbext?  And nice catch.  *That's* the case we weren't thinking of when we were trying to figure out who uses that jar.  We'll need to resolve this before we can proceed -- I'll investigate.

Reply:
Tested against pdbext, and it was fine (after restoring the test jar).